pygmt/datasets/load_remote_dataset.py

```diff
@@ -165,9 +157,8 @@ class GMTRemoteDataset(NamedTuple):
         },
     ),
     "earth_relief": GMTRemoteDataset(
```

Comment:
Perhaps we should have separate entries for `earth_relief` and `earth_gebco` since they're two different datasets.

Reply:
This is done in the commits https://github.com/GenericMappingTools/pygmt/pull/3048/commits/15f42b72064deba77870b627d93a17c98a879996 - https://github.com/GenericMappingTools/pygmt/pull/3048/commits/3ca6f323f11545695b8a54b59cc3f80fbee94b98